modules/impact/util_nodes.py

```diff
@@ -549,9 +549,8 @@ def doit(self, string, delimiter, prefix_all, postfix_all, restrict_to_tags, exc
         # some sanity checks and normalization for later processing
         if prefix_all is None: prefix_all = ""
         if postfix_all is None: postfix_all = ""
-        if restrict_to_tags is None: restrict_to_tags = ""
         if exclude_tags is None: exclude_tags = ""
-        if not isinstance(restrict_to_tags, list):
+        if restrict_to_tags is not None and restrict_to_tags != "" and not isinstance(restrict_to_tags, list):
```

Comment:
`restrict_to_tags` is always `string` and not a optional input.
So you can remove 

None check of `restrict_to_tags` and `isinstance(restrict_to_tags, list)`


```
        if restrict_to_tags != "":
            restrict_to_tags = restrict_to_tags.split(", ")
        else:
            restrict_to_tags = []
```